lang/java/java.talon

```diff
@@ -16,6 +16,10 @@ settings():
     user.code_public_variable_formatter = "PRIVATE_CAMEL_CASE"
 
 # Types Commands
+type {user.code_type}:
```

Comment:
This is already defined in `lang/tags/generic.talon`, why redefine it here? I know the pull request says you want to add a space for consistency, but what is it you're being consistent with?

Reply:
Consistent with the rest of the types and Java

Reply:
It looks like we're using the code_type list for Python and Typescript and hence the generic 'type' command which doesn't insert the space. I'd suggest we be consistent with that and don't override this here so that people get the same behaviour across languages.

What do you think? You could easily override type to add the space in your personal configuration and we can keep things more consistent in knausj.

Reply:
sure